docs/BRP-LO GBA mapping - Personen gba.csv

```diff
@@ -0,0 +1,137 @@
+categorie;element;leveren door RvIG;Beperkt zoekresultaat;Persoon GBA variant;;;;;opmerking/issue;
```

Comment:
In de mapping.md staat kolom "leveren door RvIG"uitgelegd met "Beschrijft in welke vorm het BRP gegeven geleverd wordt." Dat is alleen nodig omdat de kolomkop niet zo duidelijk is. Dat het door RvIG geleverd wordt is al duidelijk, dit is de sheet voor de GBA variant.

"Leveringsvorm"? Of "Waarde"? Of "Bewerking"?